packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -307,7 +311,12 @@ const Updates = decorate([
                 </p>
                 {state.showPackagesList && (
                   <p>
-                    <Copiable tagName='pre'>{state.packagesList}</Copiable>
+                    <Copiable tagName='pre'>
+                      <p>
```

Comment:
IMHO, it should not be encapsulated in a paragraph. because it's not a text.

Reply:
What do you propose?
I use `<p>` to have a separate block element with spaces and to not confuse it with the list of packages.

Reply:
What do you propose?
I use `<p>` to have a block element with spaces and to not confuse it with the list of packages.